### DISCUSSION TITLE
Virus detected in Deploy-Application.exe

### DISCUSSION THREAD

**Discussion by Taunusianer** [PSAppDeployToolkit/PSAppDeployToolkit#627] [General]
Hi, 
when downloading the file Deploy-Application.exe we receive an report, that a virus is detected regarding this file. Also acronis online scan reports the same. Is there an issue?
kind regards
T.

**Comment by luki1412** [comment]
No and it shouldn't report that. The code is literally available on the repository.

**Comment by Zoran-Jankov** [comment]
It is because the antivirus software is detecting a suspicious behavior in that application. It is very common for antivirus software to detect unregistered applications as malicious when they are making systems changes. Nothing to worry about...
